lambdaclass/mirra_backend
# [GH-443] [Dungeon] [Config] Create DungeonSettlementLevel config json import

## Motivation

Add DungeonSettlementLevel config json import
Closes https://github.com/lambdaclass/afk_gacha_game/issues/443

## Summary of changes

- Create json with data from https://docs.google.com/spreadsheets/d/12nSgWtkOFmL_iTCDm2LmL3DW5GAgF2jW4fqfiQJRoPs/edit#gid=2012627134
- Create import function using new upsert calls
- Change rates so that they are daily instead of second-ly 

## How to test it?

Run `Champions.Config.import_dungeon_settlement_levels_config()` (Seeds run it automatically). Check that all data is correctly written to DB and that level up works. If you want to test level up with client, you can do so with [this](https://github.com/lambdaclass/afk_gacha_game/pull/472) PR.
